### PR TITLE
Tempus:  Changes in support of pseudo-transient adjoint sensitivities for SPARC

### PR DESCRIPTION
**Pull request by etphipp**
This PR adds several changes in support of developing pseudo-transient adjoint sensitivities in SPARC.

### CODE REVIEW

**Review by ccober6 — APPROVED** [review]
LGTM

File: packages/tempus/src/Tempus_IntegratorAdjointSensitivity_impl.hpp [hunk 1]
```diff
@@ -451,7 +451,9 @@ void IntegratorAdjointSensitivity<Scalar>::
 setObserver(Teuchos::RCP<IntegratorObserver<Scalar> > obs)
 {
   state_integrator_->setObserver(obs);
-  adjoint_integrator_->setObserver(obs);
```
Comment (ccober6) [inline packages/tempus/src/Tempus_IntegratorAdjointSensitivity_impl.hpp]: @etphipp Am I missing something here?  It looks like the observer is still being passed to the adjoint integrator.

Comment (etphipp) [reply packages/tempus/src/Tempus_IntegratorAdjointSensitivity_impl.hpp]: Look closer at the change.  The original line was commented out.

Comment (ccober6) [reply packages/tempus/src/Tempus_IntegratorAdjointSensitivity_impl.hpp]: Ugh.  Missread the diff.  :(

**Review by ccober6 — APPROVED** [review]
LGTM

**Comment by etphipp** [comment]
@trilinos/framework The auto-tester is not using the latest commit to this branch in the testing for some reason.  If one looks at the CDash log for any of the failed tests, e.g., [this one](https://testing.sandia.gov/cdash/build/10461954), one can see that HEAD is at `af95cd3 Tempus:  Fix mistake in merge` which was the last commit when the PR was created.  However a new commit was added before these builds started (because I added the RETEST label after adding the commit, which kicked off the latest round of tests), but it isn't included for some reason.  Do you know what is going on here?

**Comment by jwillenbring** [comment]
@etphipp Did you happen to force push any of your changes?

**Comment by etphipp** [comment]
> @etphipp Did you happen to force push any of your changes?

No.  Just a regular push.

**Pull request merged**